models/meshmodel/core/v1alpha1/models.go

```diff
@@ -38,6 +39,9 @@ type Model struct {
 	Version     string                 `json:"version"`
 	DisplayName string                 `json:"displayName" gorm:"modelDisplayName"`
 	Category    Category               `json:"category"`
+	HostID	    uuid.UUID			   `json:"hostID"`
```

Comment:
@Freedisch please add linting at the moment you can see that stylistically code isn't good.